style.css

```diff
@@ -0,0 +1,14 @@
+header {
+  border-bottom: solid 5px rgb(15, 173, 15);
+}
+ul > * {
```

Comment:
Be careful with the universal selector, since maybe tomorrow someone add a new element that is a direct children of the ul and will be affected by it!